javascripts/next.meeting.js

```diff
@@ -0,0 +1,57 @@
+/* 
+* Dispay Next Meeting
+* i.e. calculate nearest first or third tuesday 
```

Comment:
Hmmm. It's definitely possible to fetch Meetup info [via their API](http://www.meetup.com/meetup_api/docs/2/event/#get), so I could look into that — admittedly, I'm a noob working with APIs, so I don't expect to be able to bang out a solution too quickly. But I definitely prefer something automated compared to manual entry, though. Someone is already posting to Meetup, so it would be convenient to not worry about adding the same content in two places.